container/lxd/initialisation_linux.go

```diff
@@ -293,6 +294,9 @@ func ensureDependencies(series string) error {
 	return err
 }
 
+// randomizedOctetRange is a variable for testing purposes.
+var randomizedOctetRange = func() []int { return rand.Perm(255) }
+
 // findNextAvailableIPv4Subnet scans the list of interfaces on the machine
```

Comment:
This docstring needs updating to match the new algorithm - it would be worth including a link or reference to the LXD implementation since this is based on that.